pom.xml

```diff
@@ -521,6 +523,75 @@ Import-Package: \\
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>com.diffplug.spotless</groupId>
+          <artifactId>spotless-maven-plugin</artifactId>
+          <version>${spotless.version}</version>
+          <configuration>
+            <java>
+              <eclipse>
+                <file>openhab_codestyle.xml</file>
```

Comment:
Where is this openhab_codestyle.xml file? I can't find it in the current branch or in the master...

Reply:
It is located here https://github.com/seime/openhab-core/tree/spotless/tools/codestyle/src/main/resources (PR to be merged first) and loaded as a classpath entry by the plugin.

Reply:
Correct